dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -9,22 +9,28 @@ import 'sentry_options.dart';
 class SentryStackTraceFactory {
   final SentryOptions _options;
 
-  final _absRegex = RegExp(r'^\s*#[0-9]+ +abs +([A-Fa-f0-9]+)');
-  final _frameRegex = RegExp(r'^\s*#', multiLine: true);
-
+  static final _absRegex = RegExp(r'^\s*#[0-9]+ +abs +([A-Fa-f0-9]+)');
+  static final _frameRegex = RegExp(r'^\s*#', multiLine: true);
+  static final _buildIdRegex = RegExp(r"build_id: '([A-Fa-f0-9]+)'");
+  static final _baseAddrRegex = RegExp(r'vm_dso_base: ([A-Fa-f0-9]+)');
```

Comment:
we have some overlapping implementations with https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/debug_image_extractor.dart

e.g regex, `_StackInfo` and `_DebugInfo` etc...

maybe we could merge / reuse them in whatever way you think is better to avoid more duplications

Reply:
sure, I'll update this PR deduplicate